HttpClient/FirstApp/src/app/user-post.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+import {User} from './user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserPostService {
+
+  constructor(private http: HttpClient) { }
+
+  postData(user: User): Observable<any>{
```

Comment:
```suggestion
  create(user: User): Observable<any>{
```

I think, its better to name this method "create", it seems to create user on server